src/jukebox/components/rfid/hardware/generic_nfcpy/setup.inc.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+CURRENT_USER="${SUDO_USER:-$(whoami)}"
+
+modprobe_file="/etc/modprobe.d/disable_driver_jukebox_nfcpy.conf"
+
+
+if lsmod | grep "pn533_usb"; then
+    sudo rmmod pn533_usb 2>/dev/null
+    sudo sh -c "echo 'install pn533_usb /bin/true' >> $modprobe_file"
```

Comment:
Does the entries get added multiple times if the setup is run again? 
Better check for existence